tgui/packages/tgui/interfaces/PanDEMIC.tsx

```diff
@@ -0,0 +1,370 @@
+              </Tabs.Tab>
+            ))}
+          </Tabs>
+          <StrainInformationSection
+            strain={selectedStrain}
+            strainIndex={selectedStrainIndex}
+          />
+          {selectedStrain.symptoms?.length > 0 && (
+            <StrainSymptomsSection
+              className="remove-section-bottom-padding"
+              strain={selectedStrain}
+            />
+          )}
+        </Stack>
+      </Section>
+    </Stack.Item>
+  );
+};
+
+const sum = (values: number[]) => {
+  return values.reduce((r, value) => r + value, 0);
+};
+
+const StrainSymptomsSection = (props: {
+  className?: string;
+  strain: PathogenStrain;
+}) => {
+  const { symptoms } = props.strain;
+  return (
+    <Stack.Item grow>
+      <Section title="Infection Symptoms" fill className={props.className}>
+        <Table className="symptoms-table">
+          <Table.Row>
+            <Table.Cell>Name</Table.Cell>
+            <Table.Cell>Stealth</Table.Cell>
+            <Table.Cell>Resistance</Table.Cell>
+            <Table.Cell>Stage Speed</Table.Cell>
+            <Table.Cell>Transmissibility</Table.Cell>
+          </Table.Row>
+          {symptoms.map((symptom, index) => (
+            <Table.Row key={index}>
+              <Table.Cell>{symptom.name}</Table.Cell>
+              <Table.Cell>{symptom.stealth}</Table.Cell>
+              <Table.Cell>{symptom.resistance}</Table.Cell>
+              <Table.Cell>{symptom.stageSpeed}</Table.Cell>
+              <Table.Cell>{symptom.transmissibility}</Table.Cell>
+            </Table.Row>
+          ))}
+          <Table.Row className="table-spacer" />
+          <Table.Row>
+            <Table.Cell style={{ 'font-weight': 'bold' }}>Total</Table.Cell>
+            <Table.Cell>{sum(symptoms.map((s) => s.stealth))}</Table.Cell>
+            <Table.Cell>{sum(symptoms.map((s) => s.resistance))}</Table.Cell>
+            <Table.Cell>{sum(symptoms.map((s) => s.stageSpeed))}</Table.Cell>
+            <Table.Cell>
+              {sum(symptoms.map((s) => s.transmissibility))}
+            </Table.Cell>
+          </Table.Row>
+        </Table>
+      </Section>
+    </Stack.Item>
+  );
+};
+
+const VaccineSynthesisIcons = ['flask', 'vial', 'eye-dropper'];
+
+const ResistancesSection = (props, context) => {
+  const { act, data } = useBackend<PanDEMICData>(context);
+  const { synthesisCooldown, beakerContainsVirus, resistances } = data;
+  return (
+    <Stack.Item>
+      <Section
+        title="Antibodies"
+        fill
+        buttons={!beakerContainsVirus && <CommonCultureActions />}
+      >
+        <Stack horizontal wrap>
+          {resistances.map((r, i) => (
+            <Stack.Item key={i}>
+              <Button
+                icon={VaccineSynthesisIcons[i % VaccineSynthesisIcons.length]}
+                disabled={!!synthesisCooldown}
+                onClick={() =>
+                  act('clone_vaccine', { resistance_index: i + 1 })
+                }
+                mr="0.75ex"
```

Comment:
```suggestion
                mr="0.75ex"
```
It's the first time I've ever seen such a thing
Why not a standardised value? Just a number like {0.5}

Reply:
Yeah, plus em is universally used